src/util/normalizeThought.ts

```diff
@@ -2,9 +2,17 @@ import _ from 'lodash'
 import * as pluralize from 'pluralize'
 import { REGEXP_TAGS } from '../constants'
 import emojiStrip from 'emoji-strip'
+import isAttribute from './isAttribute'
 
 /** Trims a string. */
-export const trim = (s: string) => s.replace(s.length > 0 && s.replace(/\W/g, '').length > 0 ? /\W/g : /s/g, '')
```

Comment:
The function name and comment do not match what it actually does. A developer reading this will think it is a normal trim function, and not realize it is more complicated. Please update the function name and comment to something that matches the functionality.